hydra-node/src/Hydra/Network/Etcd.hs

```diff
@@ -199,7 +201,7 @@ withEtcdNetwork tracer protocolVersion config callback action = do
       $ concat
         [ -- NOTE: Must be used in clusterPeers
           ["--name", show advertise]
-        , ["--data-dir", persistenceDir </> "etcd"]
+        , ["--data-dir", persistenceDir </> "etcd" </> hashToStringAsHex (hashWithSerialiser @SHA256 toCBOR $ BS8.pack clusterPeers)]
```

Comment:
Could: avoid `toCBOR` and use more robust `encodeUtf8` (`BS8` truncates)

```suggestion
        , ["--data-dir", persistenceDir </> "etcd" </> hashToStringAsHex (hashWith @SHA256 $ encodeUtf8 clusterPeers)]
```

Reply:
This snippet isn't correct so I will not do this right now.